src/components/RegionList/RegionListComponent.tsx

```diff
@@ -196,104 +266,94 @@ export class RegionListComponent extends React.Component<WidgetProps> {
+                        <div className="cell" style={{width: RegionListComponent.ROTATION_COLUMN_DEFAULT_WIDTH}} onDoubleClick={this.handleRegionListDoubleClick}>
                             {toFixed(region.rotation, 1)}
-                        </td>
+                        </div>
                     )}
-                </tr>
+                </div>
             );
-        });
+        };
 
         return (
             <div className="region-list-widget">
-                <HTMLTable style={{height: tableHeight}}>
-                    <thead className={appStore.darkTheme ? "dark-theme" : ""}>
-                        <tr>
-                            <th style={{width: RegionListComponent.ACTION_COLUMN_DEFAULT_WIDTH * 3}}>
-                                <Icon icon={"blank"} style={{width: 16}} />
-                                <Tooltip2 content="Import regions" position={Position.BOTTOM}>
-                                    <Icon icon={"cloud-download"} onClick={this.handleRegionImportClicked} style={{cursor: "pointer"}} />
-                                </Tooltip2>
-                                <Icon icon={"blank"} style={{width: 5}} />
-                                <Tooltip2 content="Export all regions" position={Position.BOTTOM}>
-                                    {this.validRegions.length > 1 ? <Icon icon="cloud-upload" onClick={this.handleRegionExportAllClicked} style={{cursor: "pointer"}} /> : <Icon icon="cloud-upload" style={{opacity: 0.4}} />}
-                                </Tooltip2>
-                            </th>
-                            <th style={{width: nameWidth}}>Name</th>
-                            <th style={{width: RegionListComponent.TYPE_COLUMN_DEFAULT_WIDTH}}>Type</th>
-                            <th style={{width: RegionListComponent.CENTER_COLUMN_DEFAULT_WIDTH}}>{frame.validWcs ? "Center" : "Pixel Center"}</th>
-                            {showSizeColumn && <th style={{width: RegionListComponent.SIZE_COLUMN_DEFAULT_WIDTH}}>{frame.validWcs ? "Size" : "Size (px)"}</th>}
-                            {showRotationColumn && <th style={{width: RegionListComponent.ROTATION_COLUMN_DEFAULT_WIDTH}}>P.A. (deg)</th>}
-                        </tr>
-                    </thead>
-                    <tbody className={appStore.darkTheme ? "dark-theme" : ""}>{rows}</tbody>
-                </HTMLTable>
-                <ReactResizeDetector handleWidth handleHeight onResize={this.onResize}></ReactResizeDetector>
+                <div className={classNames("region-list-table", {"bp3-dark": darkTheme})}>
+                    <FixedSizeList itemSize={25} height={25} itemCount={1} width="100%" className="list-header">
+                        {headerRenderer}
+                    </FixedSizeList>
+                    <FixedSizeList onItemsRendered={this.onListRendered} height={tableHeight - 35} itemCount={this.validRegions.length} itemSize={RegionListComponent.ROW_HEIGHT} width="100%">
```

Comment:
maybe changing `tableHeight - 35` to `tableHeight - HEADER_ROW_HEIGHT - padding * 2` is more clear